README.md

```diff
@@ -2,11 +2,6 @@
 
 "MyBudget" is a budget manager web application that allows users to keep track of their budget with different categories.
 
-
-## Link Demo
-
-[Cick here to get live demo]()
-
 ## Video Demo
 https://www.loom.com/share/122853d2b16a442295145af81cd49a0a
 
```

Comment:
- [ ] The deployed app is not working, kindly work on it.

![signup](https://user-images.githubusercontent.com/53124548/173172662-92d170d0-6ae3-46b5-bbb5-c0b07323317d.png)

- [ ] Let's put a deployed app live link in the readme file for easy access.

Reply:
Thanks for your review task is done